pkg/ssh/ssh.go

```diff
@@ -86,60 +86,3 @@ func RunAgent(sshPrivateKey string) error {
 	}
 	return nil
 }
-
-/*
```

Comment:
If we don't need this code, let's remove it altogether. git has the code if we need it back.